webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricCriteria.js

```diff
@@ -220,7 +220,8 @@ export class SakaiRubricCriteria extends RubricsElement {
                         data-criterion-id="${c.id}"
                         data-rating-id="${r.id}"
                         data-reorderable-id="${r.id}"
-                        id="rating_item_${r.id}">
+                        id="rating_item_${r.id}"
+                        draggable="${this.isLocked ? "false" : "true"}">
```

Comment:
Would that work too?
```suggestion
                        draggable="${!this.isLocked}">
```

Reply:
Not too sure. It needs to be a string and maybe that will just end up null or something. I'll try it.

Reply:
Yep, it works fine just dropping that in. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/toString

Reply:
Yeah, to me it looks like you put it just in the markup, where everything is treated as a String.